### DISCUSSION TITLE
support for "permanent" auras on enemies

### DISCUSSION THREAD

**Discussion by k0l11** [genshinsim/gcsim#801] [Ideas / Features Request]
This is relevant for simming teams against enemies like the Cryo Hypostasis.

**Comment by srliao** [comment]
Honestly.. I can't see this one happening unless/until we know exactly how these permanent auras work. For example... do they just have a very very very large durability that doesn't decay? Or do they refill every so often?

I'm hesitant on making assumptions for this because in the first place, it's a very niche situation. I'm more inclined to support adding some sort of external "attack" that applies an element to a target that is tied to a system function that can be called in gcsl.

**Comment by connorbeckerle** [comment]
I think this is more relevant now with hydro tulpa in stygian onslaught.
